fizzbuuzz.js

```diff
@@ -0,0 +1,26 @@
+
+
+function fizzbuzz(Fiz,Buz){
+var x =  Fiz.length + Buz.length;
+if(x%3==0){
+ return 'Fizz'
+}
+if(x%5==0){
+ return 'Buzz'
+}
+if(x%5==0&&x%3==0){ 
```

Comment:
This condition should come before the ones up for this program to work correctly

Reply:
@dennisja made corrections function works normally